spec/directives/extend/whitespace.hrx

```diff
@@ -49,3 +49,22 @@ Error: Expected newline.
   |            ^
   '
   input.sass 2:12  root stylesheet
+
+<===>
+================================================================================
+<===> error/after_arg/sass/input.sass
```

Comment:
Also test `@extend a\nb`.

Reply:
Tested in cd4eced4e5e1125de252490046e5eff6d498da20